mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java

```diff
@@ -202,35 +212,39 @@ private void setAttribute(Mail mail, Map<String, Calendar> calendars, Map<String
         }
     }
 
-    private Stream<ICAL> toICAL(Map.Entry<String, Calendar> entry, Map<String, byte[]> rawCalendars, MailAddress recipient, String sender) {
+    private Stream<ICALAttributeDTO> toICAL(Map.Entry<String, Calendar> entry,
+                                            Map<String, byte[]> rawCalendars,
+                                            MailAddress recipient,
+                                            MailAddress sender,
+                                            MailAddress participant) {
```

Comment:
MailAddress participant -> MailAddress replyTo

Reply:
I think it is a weaker naming.

Reply:
The fact to decide that the replyTo is the participant is a business decision that we can not take within the James component.